syntax = "proto3";

package module_b;

message ItemB {
    int32 id = 1;
}

message BasicMessageB {
    string name = 1;
}
